bloggers.json

```diff
@@ -808,5 +808,10 @@
       "rss": "http://feeds.feedburner.com/SwiatJava",
       "twitter": "@dluksza"
     }
+	    {
```

Comment:
You are missing a coma before yours entry. Also please format it the same way as the rest of file.